python/src/opendp/_extrinsics/synth/__init__.py

```diff
@@ -0,0 +1,61 @@
+from opendp.mod import Domain, Metric
```

Comment:
Not sure how this relates to the rest of it, but could this also live under `microdata`?